### ISSUE TITLE
editor.insertRight is not a function

### ISSUE THREAD

**Issue by drewcovi** [decaffeinate/decaffeinate#223]
Not sure what's going on here, running the latest node with a fresh install of decaffeinate. seems incompatible for some reason.

```
TypeError: editor.insertRight is not a function
  at /usr/local/lib/node_modules/decaffeinate/node_modules/add-variable-declarations/dist/add-variable-declarations.umd.js:377:21
  at Array.forEach (native)
  at addVariableDeclarations (/usr/local/lib/node_modules/decaffeinate/node_modules/add-variable-declarations/dist/add-variable-declarations.umd.js:376:29)
  at Function.run (/usr/local/lib/node_modules/decaffeinate/dist/decaffeinate.cjs.js:180:7)
  at /usr/local/lib/node_modules/decaffeinate/dist/decaffeinate.cjs.js:9091:28
  at Array.forEach (native)
  at runStages (/usr/local/lib/node_modules/decaffeinate/dist/decaffeinate.cjs.js:9090:10)
  at convert$1 (/usr/local/lib/node_modules/decaffeinate/dist/decaffeinate.cjs.js:9083:10)
  at ReadStream.<anonymous> (/usr/local/lib/node_modules/decaffeinate/dist/decaffeinate.cjs.js:9019:19)
  at emitNone (events.js:91:20)
  at ReadStream.emit (events.%
 ```

**Comment by eventualbuddha** [comment]
This is most likely due to me updating add-variable-declaration to use magic-string 0.13.1 but npm using the same version for both. Probably requires pinning to the exact version of magic-string.

**Comment by eventualbuddha** [comment]
Fixed in v2.10.1.
